docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,790 @@
+* Fixes multi terms fields validation in *Lens* {kibana-pull}126618[#126618]
+* Make Embeddable resilient when toggling actions in *Lens* {kibana-pull}126558[#126558]
+* Make graph app resilient to no fields or missing data views {kibana-pull}126441[#126441]
+* Fixes Formula to Quick functions does not preserve custom formatting in *Lens* {kibana-pull}124840[#124840]
+* Inspector displays only visible content {kibana-pull}124677[#124677]
+* Coloring tooltips in Heatmap are not properly positioned in *Visualize* {kibana-pull}124507[#124507]
+* Adds rison helper and URL encoding for drilldown urls in *TSVB* {kibana-pull}124185[#124185]
+
+Machine Learning::
+* Fixes alignment of Anomaly Explorer swim lane annotations label on Firefox {kibana-pull}130274[#130274]
+* Fixes Single Metric Viewer chart failing to load if no points during calendar event {kibana-pull}130000[#130000]
+* Fixes Single Metric Viewer for jobs that hasn't been run {kibana-pull}129063[#129063]
+* Data Frame Analytics: Fix Outlier detection results exploration color legend display {kibana-pull}129058[#129058]
+* Fixes new AD job from saved search with no query filter {kibana-pull}129022[#129022]
+* Fixes DFA map saved object sync warning {kibana-pull}128876[#128876]
+* Fixes get_filter endpoint {kibana-pull}128238[#128238]
+* Adds error toast to Data visualizer when using unpopulated time field {kibana-pull}127196[#127196]
+* Adds data recognizer module config cache {kibana-pull}126338[#126338]
+
+Management::
+* Transforms: Fix to not pass on default values in configurations {kibana-pull}129091[#129091]
+* Encode + sign in ISO8601 time range in query {kibana-pull}126660[#126660]
+
+Maps::
+* Fixes lens region map visualization throws a silent error {kibana-pull}129608[#129608]
+* Fixes double click issue when deleting a shape {kibana-pull}124661[#124661]
+
+Monitoring::
+* Exclude Malwarescore + Ransomware EP alerts from DRule telemetry {kibana-pull}130233[#130233]
+* Rename "APM & Fleet Server" to "Integrations Server" {kibana-pull}128574[#128574]
+* Fixes sorting by node status on nodes listing page {kibana-pull}128323[#128323]
+
+Observability::
+* Service environment should be selected when you edit the agent configuration {kibana-pull}129929[#129929]
+* Adds migration to include synthetics and heartbeat indices on 8.2.0 {kibana-pull}129510[#129510]
+* Rules summary on the Alerts view is not showing the count of rules {kibana-pull}129052[#129052]
+* Fixes shadow for overview panels {kibana-pull}128878[#128878]
+* Ensure rum_allow_origins setting only saves valid YAML strings {kibana-pull}128704[#128704]
+* Standardize NOW as startedAt from executor options {kibana-pull}128020[#128020]
+* Fixes synthetics recorder file upload {kibana-pull}127614[#127614]
+* Service maps popover detail metrics are aggregates over all transaction types {kibana-pull}125580[#125580]
+
+Platform::
+* Duplicated datatable available as inspector data for Heatmap chart {kibana-pull}126786[#126786]
```

Comment:
The title for the original PR here was confusingly worded. This isn't my team (cc @flash1293), but it looks like there are really two things that were fixed here:
- We fixed an issue that would cause duplicate data in the inspector datatable in Lens for Heatmap charts
- We fixed an issue that would cause stale data to persist in the inspector datatable when layers were removed